src/main/java/edu/kis/powp/jobs2d/command/visitor/VisitorCounter.java

```diff
@@ -11,8 +11,8 @@
 {
     private int counter = 0;
 
-    private double currentX = 0;
-    private double currentY = 0;
+    double currentX = 0;
+    double currentY = 0;
```

Comment:
What is it for? Fields seems seem not used and not useful to be exposed within the package.

Reply:
fixed in next commit